src/v0/safes.ts

```diff
@@ -156,4 +169,20 @@ export class Safes {
       }
     })
   }
+
+  book(body: BookRequestBody): Promise<SafeResponse> {
```

Comment:
Because it's a post request I would rename the handler `setBook` to imply what it does.

Reply:
Hmm, I wanted to be consistent with the other namings, that are also just one verb like "get" or "create". So "book" is actually following that logic.

Reply:
Ohh, "book" is a verb in this case. I read it as a noun.